pkg/webhooks/handlers/filter.go

```diff
@@ -26,7 +26,7 @@ func (inner AdmissionHandler) WithSubResourceFilter(subresources ...string) Admi
 }
 
 func filtered(ctx context.Context, logger logr.Logger, request AdmissionRequest, message string, keysAndValues ...interface{}) AdmissionResponse {
-	logger.V(2).Info(message, keysAndValues...)
+	logger.V(1).Info(message, keysAndValues...)
```

Comment:
Do we need it at level 1 or 4?

Reply:
> Do we need it at level 1 or 4?

I think level 1 reduces verbosity , higher level means more verbosity.

Reply:
The default verbosity level is 2. Setting this value to 4 means that this message will be displayed at log level 4 or higher.

Reply:
> The default verbosity level is 2. Setting this value to 4 means that this message will be displayed at log level 4 or higher.

Yes, but the issue mentioned the kyverno reports are flooded needlessly so i reduced the default to 1.  Do I need to make some changes?